en/docs/deploy/change-the-hostname.md

```diff
@@ -114,85 +98,61 @@ This section guides you through changing the hostname of WSO2 Identity Server.
+By doing that, you do not need to touch the key pair, and any other operations performed using the same key pair, such as encryption, will not be affected.
 
 **Option 2**
 
-Create a new keystore with the instructions for the new hostname and then use that keystore for
-SSL/TLS by changing Tomcat connector configuration as described in the
-[Configure Keystores]({{base_path}}/deploy/security/configure-keystores-in-wso2-products/) topic. This approach
-separates the keystores. The secondary keystore with the new hostname
-will only be used for Tomcat SSL/TLS communication, while the primary
-one is used for all other operations (encryption, etc.). By doing this,
-you can make sure that the existing encrypted data is not affected.
+Create a new keystore with the instructions for the new hostname and then use that keystore for SSL/TLS by changing the Tomcat connector configuration as described in the [Configure Keystores]({{base_path}}/deploy/security/configure-keystores-in-wso2-products/) topic. This approach separates the keystores. 
+The secondary keystore with the new hostname will only be used for Tomcat SSL/TLS communication, while the primary one will be used for all other operations (encryption, etc.). By doing this, you can ensure the existing encrypted data is not affected.
 
 !!! note
-    
-    If you have chosen **Option 2**, you need to maintain and secure the two
-    keystores. Hence, **Option 1 is recommended**, unless you
-    prefer separation of keys used in SSL/TLS communication and internal
-    data encryption, signing, etc.
-    
+    If you have chosen **Option 2**, you need to maintain and secure the two keystores. Hence, **Option 1** is recommended unless you prefer separation of keys used in SSL/TLS communication and internal data encryption, signing, etc.
 
-Once this is done, you need to change all `localhost` references. The following points would be the main places where 
-the references should be changed.
+Once this is done, you need to change all `localhost` references. The following points would be where the references should be changed.
 
-- The configurations in the `<IS_HOME/repository/conf/deployment.toml` file that references localhost.
+- The configurations in the `<IS_HOME>/repository/conf/deployment.toml` file reference localhost.
 - The entity ID that is available in the resident identity provider configurations.
+
+    - If you have added the proxy port as `443` to the `<IS_HOME>/repository/conf/deployment.toml` using the below configs, use `https://domain/oauth2/token` as the entity ID value.
+
+        ```toml
+        [transport.https.properties]
+        proxyPort = 443
+        ```
+
+    - Else, use `https://domain:<PORT>/oauth2/token` as the entity ID value.
```

Comment:
@mifrazmurthaja @ShanChathusanda93

Reply:
Shall we mention it as, "Identity Provider Entity ID", under the section, Inbound Authentication Configuration-> OAuth2/OpenID Connect Configuration, instead of just Entity ID? Also, since the domain is also a placeholder, shall we add it as `<DOMAIN>`?